RELEASE.md

```diff
@@ -45,6 +45,15 @@ If a bug fix got accidentally merged into master after non-bug-fix changes in ma
 
 Maintaining the release branches for older minor releases happens on a best effort basis.
 
+### Show that a release is in progress
+
+This helps ongoing PRs to get their changes in the right place, and to consider whether they need cherry-picked.
+
+1. Make a PR to update `CHANGELOG.md` on master
```

Comment:
💯  agree on this! Thanks!

Reply:
Good idea. I still think it can also be done on release branch, and just merge release branch early and more often, but this works too.